index.js

```diff
@@ -17,6 +17,12 @@ var FindParent = {
     return this.byMatcher(element, function(el) {
       return el.classList.contains(className);
     });
+  },
+
+  byAttribute: function(element, attName) {
+    return this.byMatcher(element, function(el) {
+      return el.hasAttribute(attName);
```

Comment:
Perhaps it should be called `withDataAttribute(element, dataName)` or something.